edx_exams/apps/lti/signals/tests/test_handlers.py

```diff
@@ -8,9 +8,9 @@
 from ddt import ddt, idata, unpack
 from django.test import TestCase
 
-from edx_exams.apps.api.test_utils.factories import UserFactory
 from edx_exams.apps.core.models import Exam, ExamAttempt, ProctoringProvider
 from edx_exams.apps.core.statuses import ExamAttemptStatus
+from edx_exams.apps.core.test_utils.factories import UserFactory
```

Comment:
It made more sense for this util to live in core since we really wouldn't want the lti/core apps to depend on the api app.